### PR TITLE
Feature/277: Fix reloading workspace mod files

### LINKED ISSUE #277: import PTcell and ‘compile mod’ - error (already compiled)

### PR DESCRIPTION
**Pull request by filippomc**
Implemented fix 1: force neuron.nrn_dll_loaded from #277 

Closes #277

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,4 +28,7 @@ RUN python install.py ${BUILD_ARGS}
 
 WORKDIR ${INSTALLATION_FOLDER}
 
+RUN pip install -r requirements-test.txt
+RUN pip install -e .
```
Comment (lrebscher) [inline Dockerfile:32]: Why install in edit mode here?

File: netpyne_ui/mod_utils.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    else:
+        return True
+
+
+def compileModMechFiles(compileMod, modFolder):
+    # Create Symbolic link
+    if compileMod:
+        modPath = os.path.join(str(modFolder), "x86_64")
+
+        subprocess.call(["rm", "-r", modPath])
```
Comment (lrebscher) [inline netpyne_ui/mod_utils.py:33]: not sure if that's a case but this might throw an Exception when modPath is not present

**Review by lrebscher — APPROVED** [review]
lgtm, thx for setting up unit tests!

File: Dockerfile (hunk 1 above)
Comment (filippomc) [reply Dockerfile:32]: it's an easy way to run the tests and have correct references. I also think it's fine to have the "main" liibrary coming form the main files

File: netpyne_ui/mod_utils.py (hunk 2 above)
Comment (filippomc) [reply netpyne_ui/mod_utils.py:33]: I only moved that code. You're right but probably subprocess fails without throwing an error

**Pull request merged**